Dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM golang:1.20 as builder
-WORKDIR /netreap
-COPY . /netreap
+FROM --platform=${BUILDPLATFORM} golang:1.20.5 as builder
```

Comment:
I don't think you need the `--platform` argument since buildx takes care of that for you, at least in CI